contracts/src/TokenRemote/TokenRemote.sol

```diff
@@ -32,65 +32,73 @@ import {TokenScalingUtils} from "../utils/TokenScalingUtils.sol";
  * @custom:security-contact https://github.com/ava-labs/avalanche-interchain-token-transfer/blob/main/SECURITY.md
  */
 abstract contract TokenRemote is ITokenRemote, TeleporterOwnerUpgradeable, SendReentrancyGuard {
-    /// @notice The blockchain ID of the chain this contract is deployed on.
-    bytes32 public blockchainID;
-
-    /// @notice The blockchain ID of the TokenHome instance this contract receives tokens from.
-    bytes32 public tokenHomeBlockchainID;
-
-    /// @notice The address of the TokenHome instance this contract receives tokens from on the {tokenHomeBlockchainID}.
-    address public tokenHomeAddress;
-
-    /**
-     * @notice The number of decimal places in the denomination of the home
-     * token.
-     * @dev Used to derive tokenMultiplier and multiplyOnRemote.
-     */
-    uint8 public homeTokenDecimals;
-
-    /**
-     * @notice The number of decimal places in the denomination of the remote token.
-     * @dev Used to derive tokenMultiplier and multiplyOnRemote.
-     */
-    uint8 public tokenDecimals;
-
-    /**
-     * @notice tokenMultiplier allows this contract to scale the number of tokens it sends/receives to/from
-     * its token TokenHome instance.
-     *
-     * @dev This is used to normalize the number of decimal places between the token home asset and the
-     * token remote asset. It is derived from home and remote token decimals values passed in the constructor.
-     */
-    uint256 public tokenMultiplier;
-
-    /**
-     * @notice If {multiplyOnRemote} is true, the home token amount will be multiplied by {tokenMultiplier} when tokens
-     * are transferred from the home into this remote, and divided by {tokenMultiplier} when tokens are transferred from
-     * this remote back to its home.
-     * If {multiplyOnRemote} is false, the home token amount value will be divided by {tokenMultiplier} when tokens
-     * are transferred from the home into this remote, and multiplied by {tokenMultiplier} when tokens are transferred
-     * from this remote back to its home.
-     */
-    bool public multiplyOnRemote;
-
-    /**
-     * @notice Initial reserve imbalance that the token for this TokenRemote instance starts with. The home contract must
-     * collateralize a corresonding amount of home tokens before tokens can be minted on this contract.
-     */
-    uint256 public initialReserveImbalance;
+    // solhint-disable private-vars-leading-underscore
+    /// @custom:storage-location erc7201:avalanche-ictt.storage.TokenRemote
+    struct TokenRemoteStorage {
+        /// @notice The blockchain ID of the chain this contract is deployed on.
+        bytes32 _blockchainID;
+        /// @notice The blockchain ID of the TokenHome instance this contract receives tokens from.
```

Comment:
Also spacing nit here, what do think of including the empty newline between these state variables like we had before? Makes it a bit easier to read IMO

Reply:
it's getting autoformatted being the types are in a struct, going to leave it for now.